lib/cisco_node_utils/cmd_ref/snmp_user.yaml

```diff
@@ -1,49 +1,67 @@
 # snmp_user
 ---
-_exclude: [ios_xr]
-
 _template:
   multiple: true
```

Comment:
consider adding the get_commands to the template, since they seem to be the same in most of the attributes (just override the one or two that are different)?